packages/taco/src/conditions/base/shared.ts

```diff
@@ -1,11 +1,19 @@
 import { z } from 'zod';
 
 import { ETH_ADDRESS_REGEXP, USER_ADDRESS_PARAM } from '../const';
+import { CONTEXT_PARAM_PREFIX } from '../context/context';
+
+export const ContextParamSchema = z.string().startsWith(CONTEXT_PARAM_PREFIX);
```

Comment:
If using the regex, can we do:
```suggestion
export const ContextParamSchema = z.string().regex(...);
```